app/models/work_version.rb

```diff
@@ -117,7 +117,10 @@ def label(id)
 
   validates :rights,
             presence: true,
-            inclusion: { in: WorkVersion::Licenses.ids },
+            inclusion: {
+              in: WorkVersion::Licenses.ids,
+              allow_nil: true # Avoid duplicating the above presence validation
```

Comment:
Rights can't be nil. Are you saying we can take out `presence: true`?

Reply:
No, the allow_nil is on the inclusion validation only. 

Previously if you had a nil value in the rights you would get two different validation errors:
- Rights can't be blank
- Rights is not included in the list [Because nil is not a member of the list]

That's super confusing and unintuitive. If you pop `allow_nil: true` onto the inclusion validation, it _still_ will not allow rights to be nil, because you have the presence validation on there, but it will not duplicate the two error messages

Reply:
Before: 
![Screen Shot 2020-10-15 at 1 23 56 PM](https://user-images.githubusercontent.com/14730/96164774-b57d7900-0ee9-11eb-8c93-e86ba4d8d0aa.png)

After:
![Screen Shot 2020-10-15 at 1 23 42 PM](https://user-images.githubusercontent.com/14730/96164785-b9110000-0ee9-11eb-9ca8-450c0c8b2ce4.png)